examples/disaggregated/slurm/benchmark/submit.py

```
@@ -736,12 +749,17 @@ def submit_job(config, log_dir, dry_run):
 
     # Generate wait server command (use script_dir for wait_server.sh)
+    server_health_timeout = get_server_health_timeout(env_config)
+    wait_server_script = os.path.join(script_dir, 'wait_server.sh')
+    wait_server_command = (
+        f"bash {wait_server_script} {disagg_server_hostname} "
+        f"{disagg_server_port} {server_health_timeout}")
     cmd = [
         "srun -l",
         f"--container-name={container_name}",
         f"--container-mounts={container_mount_str}",
         f"--mpi=pmix --overlap -N 1 -n 1",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)